frontend/src/pages/modelRegistry/screens/useRegisteredModelUrl.ts

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { ModelRegistryContext } from '~/concepts/modelRegistry/context/ModelRegistryContext';
+import { RegisteredModel } from '~/concepts/modelRegistry/types';
+
+const useRegisteredModelUrl = (rm: RegisteredModel): string => {
```

Comment:
Yeah, good call to have a utils for that, we might refactor it into another place but it's a great strat.